conf/clusters/prod.json

```diff
@@ -2,71 +2,49 @@
   "id": "prod",
   "data_sources": {
     "kinesis": {
-      "prefix_cluster1_stream_alert_kinesis": {
-        "logs": [
-          "cloudwatch",
-          "ghe",
-          "osquery"
-        ]
-      }
+      "prefix_cluster1_stream_alert_kinesis": [
```

Comment:
these should should be `prefix_cluster1_streamalert` (format is `<prefix>_<cluster>_streamalert`)